### DISCUSSION TITLE
Mongodb realm with my own backend as app MERN

### DISCUSSION THREAD

**Discussion by nynrathod** [mongodb/mongoid#5616] [Q&A]
Hi,
As I understood that mongodb realm work as backend as a service, but I want to use my own backend as express js and frontend with react or any other and want to use realm as local database + mongodb as online database, can I use like this, if yes then how?

**Comment by alexbevi** [comment]
Hey @nynrathod, this isn't really a Mongoid question and I see you've cross-posted from a [community forums post](https://www.mongodb.com/community/forums/t/mongodb-realm-with-my-own-backend/225317) as well, so I'm going to closet his out so our community team can address this in the appropriate place.
